euth_wagtail/assets/scss/components/_projects.scss

```diff
@@ -30,16 +30,26 @@
         margin: 0;
     }
 
+    &.finished .project-tile-image-link {
+        border: solid 5px $brand-brand2;
+    }
+
+    &.finished .project-tile-badge {
+        background-color: $brand-brand2;
+        color: $gray-darker;
+        text-transform: uppercase;
+    }
+
     &.public .project-tile-image-link {
         border: solid 5px $brand-brand3;
     }
 
     &.running-out .project-tile-badge {
-        background-color: #c8423a;
+        background-color: $brand-danger;
     }
```

Comment:
Can't this be removed when you just use `.euth-badge.running-out` in the markup as classes for the badges? The same goes for the other styles of course (`.finished`?)